assets/css/abstracts/_colors.scss

```diff
@@ -18,6 +18,7 @@ $no-stock-color: $alert-red;
 $low-stock-color: $alert-yellow;
 $in-stock-color: $alert-green;
 
+$placeholder-color: var(--global--color-primary, $gray-200);
```

Comment:
Correct me if I'm wrong, but `--global--color-primary` is not a variable we will get from Gutenberg but [something specific to the theme](https://github.com/WordPress/twentytwentyone/blob/trunk/assets/sass/01-settings/global.scss#L81), no?

In that case, I think we should only use that variable inside ` .theme-twentytwentyone { ... }` selectors.